include/GafferSceneUI/SceneView.h

```diff
@@ -139,6 +140,8 @@ class GAFFERSCENEUI_API SceneView : public GafferUI::View
 		std::unique_ptr<Grid> m_grid;
 		class Gnomon;
 		std::unique_ptr<Gnomon> m_gnomon;
+		class FPS;
+		std::unique_ptr<FPS> m_fps;
```

Comment:
This is an ABI break, so we should target `main` rather than `0.61_maintenance`.